python/lsst/ip/isr/isrStatistics.py

```diff
@@ -504,5 +554,171 @@ def copyCalibDistributionStatistics(self, inputExp, **kwargs):
                         key = f"LSST CALIB {calibType.upper()} {amp.getName()} DISTRIBUTION {pct}-PCT"
                         ampStats[key] = metadata.get(key, np.nan)
                 outputStats[amp.getName()] = ampStats
+        return outputStats
+
+    def measureBiasShifts(self, inputExp, overscanResults):
+        """Measure number of bias shifts from overscan data.
+
+        Parameters
+        ----------
+        inputExp : `lsst.afw.image.Exposure`
+            Exposure to measure.
+        overscans : `list` [`lsst.pipe.base.Struct`]
+            List of overscan results.  Expected fields are:
+
+            ``imageFit``
+                Value or fit subtracted from the amplifier image data
+                (scalar or `lsst.afw.image.Image`).
+            ``overscanFit``
+                Value or fit subtracted from the overscan image data
+                (scalar or `lsst.afw.image.Image`).
+            ``overscanImage``
+                Image of the overscan region with the overscan
+                correction applied (`lsst.afw.image.Image`). This
+                quantity is used to estimate the amplifier read noise
+                empirically.
+
+        Returns
+        -------
+        outputStats : `dict` [`str`, [`dict` [`str`,`float]]
+            Dictionary of measurements, keyed by amplifier name and
+            statistics segment.
+
+        Notes
+        -----
+        Based on eop_pipe implementation:
+        https://github.com/lsst-camera-dh/eo_pipe/blob/main/python/lsst/eo/pipe/biasShiftsTask.py  # noqa: E501 W505
+        """
+        outputStats = {}
+
+        detector = inputExp.getDetector()
+        for amp, overscans in zip(detector, overscanResults):
+            ampStats = {}
+            # Add fit back to data
+            rawOverscan = overscans.overscanImage.image.array + overscans.overscanFit
+
+            # Collapse array, skipping first three columns
+            rawOverscan = np.mean(rawOverscan[:, 3:], axis=1)
```

Comment:
Should this also be configurable?

Reply:
This is now `biasShiftColumnSkip`, and the already existing config renamed `biasShiftRowSkip`.